libtcmu_config.h

```diff
@@ -31,7 +31,8 @@ struct tcmu_config {
  *    5: DEBUG SCSI CMD
  */
```

Comment:
Can you add the new loglevel here and change the logging levels to 6

Reply:
will fix it.
Thanks.